evap/evaluation/models.py

```diff
@@ -252,10 +256,14 @@ def evaluation_begin(self):
     def evaluation_end(self):
         pass
 
-    @transition(field=state, source='evaluated', target='reviewed', conditions=[is_fully_checked])
+    @transition(field=state, source='evaluated', target='reviewed', conditions=[is_fully_reviewed])
     def review_finished(self):
         pass
 
+    @transition(field=state, source='reviewed', target='evaluated', conditions=[is_not_fully_reviewed])
```

Comment:
how can a course be in the state reviewed and at the same time not be fully reviewed?